realm/realm-library/src/main/java/io/realm/RealmCache.java

```diff
@@ -453,4 +474,23 @@ private static int storeColumnIndices(ColumnIndices[] array, ColumnIndices colum
         array[candidateIndex] = columnIndices;
         return candidateIndex;
     }
+
+    public RealmConfiguration getConfiguration() {
+        return configuration;
+    }
+
+    public ColumnIndices[] getTypedColumnIndicesArray() {
+        return typedColumnIndicesArray;
+    }
+
+    /**
+     * @return the total global ref count.
+     */
+    private int getTotalGlobalRefCount() {
+        int totalRefCount = 0;
+        for (RealmCacheType type : RealmCacheType.values()) {
```

Comment:
Yeah, right, i will change that to iterate on `refAndCountMap.values()` instead. Thanks!